integreat_cms/static/src/js/media-management/component/edit-directory-sidebar.tsx

```diff
@@ -105,38 +112,59 @@ const EditDirectorySidebar = ({
-                                        disabled={isLoading || directory.numberOfEntries !== 0}
-                                        onClick={showConfirmationPopupAjax}
-                                        onaction-confirmed={() => document.getElementById("delete-directory").click()}>
-                                        <Trash2 class="mr-2 inline-block h-5" />
-                                        {mediaTranslations.btn_delete_directory}
-                                    </button>
-                                )}
+                            <div>
+                                <div class="flex flex-col gap-4">
+                                    {(isDirectoryNameEditable || changedDirectory.isHidden !== directory.isHidden) && (
+                                        <button
+                                            title={mediaTranslations.btn_change_directory}
+                                            class="btn"
+                                            type="submit"
+                                            disabled={isLoading}>
+                                            <Save class="mr-1 inline-block h-5" />
+                                            {mediaTranslations.btn_change_directory}
+                                        </button>
+                                    )}
+                                    {canDeleteDirectory && (
+                                        <button
+                                            title={`${
+                                                directory.numberOfEntries === 0
+                                                    ? mediaTranslations.btn_delete_directory
+                                                    : mediaTranslations.btn_delete_empty_directory
+                                            }`}
+                                            className={cn("btn", {
+                                                "btn-red": !isLoading && directory.numberOfEntries === 0,
+                                            })}
+                                            data-confirmation-title={mediaTranslations.text_dir_delete_confirm}
+                                            data-confirmation-subject={directory.name}
+                                            disabled={isLoading || directory.numberOfEntries !== 0}
+                                            onClick={showConfirmationPopupAjax}
+                                            onaction-confirmed={() =>
+                                                document.getElementById("delete-directory").click()
+                                            }>
+                                            <Trash2 class="mr-2 inline-block h-5" />
+                                            {mediaTranslations.btn_delete_directory}
+                                        </button>
+                                    )}
+                                </div>
```

Comment:
```suggestion
                            <div class="flex flex-col gap-4">
                                {(isDirectoryNameEditable || changedDirectory.isHidden !== directory.isHidden) && (
                                    <button
                                        title={mediaTranslations.btn_change_directory}
                                        class="btn"
                                        type="submit"
                                        disabled={isLoading}>
                                        <Save class="mr-1 inline-block h-5" />
                                        {mediaTranslations.btn_change_directory}
                                    </button>
                                )}
                                {canDeleteDirectory && (
                                    <button
                                        title={`${
                                            directory.numberOfEntries === 0
                                                ? mediaTranslations.btn_delete_directory
                                                : mediaTranslations.btn_delete_empty_directory
                                        }`}
                                        className={cn("btn", {
                                            "btn-red": !isLoading && directory.numberOfEntries === 0,
                                        })}
                                        data-confirmation-title={mediaTranslations.text_dir_delete_confirm}
                                        data-confirmation-subject={directory.name}
                                        disabled={isLoading || directory.numberOfEntries !== 0}
                                        onClick={showConfirmationPopupAjax}
                                        onaction-confirmed={() => document.getElementById("delete-directory").click()}>
                                        <Trash2 class="mr-2 inline-block h-5" />
                                        {mediaTranslations.btn_delete_directory}
                                    </button>
                                )}
```
Isn't this just one superfluous nested `<div>`?

Reply:
Good catch! I think this was a remain from something we had tried